modules/ROOT/pages/getting-started.adoc

```diff
@@ -48,28 +49,32 @@ qemu-kvm -m 2048 -cpu host -nographic -snapshot \
 qemu-img create -f qcow2 -b fedora-coreos-qemu.qcow2 my-fcos-vm.qcow2
 qemu-kvm -m 2048 -cpu host -nographic \
 	-drive if=virtio,file=my-fcos-vm.qcow2 \
-	-fw_cfg name=opt/com.coreos/config,file=path/to/example.ign
+	-fw_cfg name=opt/com.coreos/config,file=path/to/example.ign \
+	-nic user,model=virtio,hostfwd=tcp::2222-:22
 ----
 +
 .Example launching FCOS with virt-install
 [source, bash]
 ----
 virt-install --connect qemu:///system -n fcos -r 2048 --os-variant=fedora31 --import \
 	--graphics=none --disk size=10,backing_store=/path/to/fedora-coreos-qemu.qcow2 \
-	--qemu-commandline="-fw_cfg name=opt/com.coreos/config,file=/path/to/example.ign"
+	--qemu-commandline="-fw_cfg name=opt/com.coreos/config,file=/path/to/example.ign" \
+	--qemu-commandline="-device virtio-net,netdev=user.0 -netdev user,id=user.0,hostfwd=tcp::2222-:22"
 ----
 
 NOTE: When using `virt-install`, you must specify absolute paths to the image and the Ignition file.
 
 TIP: If running with SELinux enabled, you may need to change the label of the Ignition file to allow access: `chcon -t svirt_home_t path/to/example.ign`
 
-Once the VM has finished booting, its IP addresses will appear on the serial console. You can then SSH into the FCOS VM as the `core` user:
+Once the VM has finished booting, its IP addresses will appear on the serial console. You can then SSH into the FCOS VM as the `core` user (replacing `localhost` and `2222` with the IP address and port of the VM, if different values from the suggested values mentioned earlier were chosen):
 
 [source, bash]
 ----
-ssh core@<ip address>
+ssh core@localhost -p 2222
 ----
 
+NOTE: With QEMU/libvirt, the IP addresses are assigned by the QEMU usermode networking backend, and will not be directly visible from your host machine. However, via host port forwarding through `localhost` (shown above), the machine is accessible.
```

Comment:
Hmm, feels like we can drop this paragraph given that it's explained in the sentence just before the `qemu-kvm` command-line?

Reply:
Though I think we should have a note here to mention that the address shown in the console in qemu is not reachable from the host, and SSH port is forwarded to an higher port on localhost instead.

Reply:
:+1: I condensed this note a bit, and kept the explanation at the above explanation (mentioning briefly the IP address not being reachable).